### ISSUE TITLE
Export to CSV works but not to XML

### ISSUE THREAD

**Issue by lollingtonbear** [xeals/signal-back#70]
When exporting to CSV the backup is read and the data is exported successfully. But when switching to XML format for the output I get a totally empty (zero size) backup.xml file.

There are no error messages. The same result on Windows and Linux

- Operating system:
tried on Linux Mint and Windows 10

- Build version (`signal-back --version`):
On Windows:
signal-back v0.1.7-alpha.2
proto commit: d6610f0
On Linux:
signal-back v0.1.7-alpha.2-dirty
proto commit: d6610f0

# Detailed description
As above.

# Full relevant output

**Comment by humblenewbie** [comment]
Have you have find a way to resolve this?

It's the same for me - I'm on a Mac (Big Sur).
The XML-file is empty (zero size). The CSV-file also works and the image/video extraction also works.

Seems the 'sender' phone numbers are lost in the CSV file - is there any way to keep the phone numbers in the extraction?

**Comment by francwalter** [comment]
Yes, xml is not working. I get a xml-file but only the skeleton, not content (<smses count="0"></smses>), so xml-export is broken. csv works though.
I don't know if phone numbers are even kept in xml output. I tried raw (gives sql-statments) but even here, no phone numbers to find 😢 
Will this be fixed?
Thank.

**Comment by Lemonsavery** [comment]
@francwalter +1
